### PR TITLE
Added animationDurationMS and updateCustomOptions to the IChartOptions

### PR DESCRIPTION
**Pull request by violetVo**
Code flow: http://codeflow/extensions/launcher.html?server=https:%2f%2fmseng.visualstudio.com%2f&projectId=96a62c4a-58c2-4dbb-94b6-5979ebc7f2af&reviewId=559435&projectshortname=AppInsights

animationDurationMS:
image

updateCustomOptions:
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,13 +53,15 @@ chartHelper.draw(queryResultData, chartOptions);
...
 | numberFormatter        | Function<br/>(numberValue: number): string | Callback that is used to format number values both in the axis and the tooltip<br/>Callback inputs:<br/>&nbsp;&nbsp;&nbsp;&nbsp;numberValue - The original number<br/>Callback return value:<br/>&nbsp;&nbsp;&nbsp;&nbsp;The string represents the display value of the numberValue |If not provided - the default formatting will apply |
 | xAxisTitleFormatter    | Function<br/>(xAxisColumn: IColumn): string | Callback that is used to get the xAxis title<br/>Callback inputs:<br/>&nbsp;&nbsp;&nbsp;&nbsp;[IColumn](#IColumn) - The x-axis column<br/>Callback return value:<br/>&nbsp;&nbsp;&nbsp;&nbsp;The desired x-axis title | If not provided - the xAxis title will be the xAxis column name |
 | yAxisTitleFormatter    | Function<br/>(yAxisColumns: IColumn[]): string | Callback that is used to get the yAxis title<br/>Callback inputs:<br/>&nbsp;&nbsp;&nbsp;&nbsp;[IColumn[]](#IColumn) - The y-axis columns<br/>Callback return value:<br/>&nbsp;&nbsp;&nbsp;&nbsp;The desired y-axis title | If not provided - the yAxis title will be the first yAxis column name |
```
Comment (ischrei) [inline README.md:63]: Can you add screenshots here?

File: README.md [hunk 2]
```diff
@@ -53,13 +53,15 @@ chartHelper.draw(queryResultData, chartOptions);
...
+| updateCustomOptions    | Function<br/>(originalOptions: any): void | Callback that is called to allow altering the options of the external charting library before rendering the chart <br/> Used to allow flexibility and control of the external charting library <br/> <b>USE WITH CAUTION</b> changing the original options might break the functionality / backward compatibility <br/> when using a different IVisualizer or upgrading the charting library <br/> Validating the updated options is the user's responsibility <br/> For official chart options - please make contribution to the base code <br/> Callback inputs:<br/>&nbsp;&nbsp;&nbsp;&nbsp;originalOptions - The custom charting options that are given to the external charting library<br/> | |
```
Comment (ischrei) [inline README.md:64]: `Not sure how to mark specific chars here.. but add dots at the end of the sentences.

File: src/visualizers/highcharts/highchartsVisualizer.ts [hunk 3]
```diff
@@ -175,7 +180,7 @@ export class HighchartsVisualizer implements IVisualizer {
         if(finishChartAnimationCallback) {
             setTimeout(() => {
                 finishChartAnimationCallback(options.chartInfo);
-            }, ANIMATION_DURATION_MS + 200);
```
Comment (ischrei) [inline src/visualizers/highcharts/highchartsVisualizer.ts:178]: can this be null? can the animationDurationMS  value in this place can be undefined?

File: README.md (hunk 1 above)
Comment (violetVo) [reply README.md:63]: See in code flow.

File: src/visualizers/highcharts/highchartsVisualizer.ts (hunk 3 above)
Comment (violetVo) [reply src/visualizers/highcharts/highchartsVisualizer.ts:178]: No. The default is 1000 so if it wasn't provided by the user, the value will be 1000.

**Pull request merged**